exercisePPJ_06_I/FunctionPPJ_06_I.java

```diff
@@ -0,0 +1,52 @@
+public class FunctionPPJ_06_I {
+	
+	private double binaryWrt[] = {0,0,0,0,0,0,0,0};
+	private double wrtDouble = 0;
+	
+	public void setFunction(double wrtD) {
+		wrtDouble = wrtD;
+	}
+	
+	public void getFunction() {
```

Comment:
It's obvious it's function. 
Please, consider some more meaningful name.

Reply:
@MatDls 
ok, i will change to more pointing to destination